config.toml

```diff
@@ -223,6 +223,60 @@ memo_prefix = ''
 #   ['transfer', 'channel-0'],
 # ]
 
+# Specify custom ICS23 proof-specs for the chain (serialized as JSON).
+# Default: [`ProofSpecs::cosmos()`](modules/src/core/ics23_commitment/specs.rs)
+proof-specs = '''
```

Comment:
Is there a more operator-friendly manner to specify this? I tried to give an alternative suggestion in #1630 .